### ISSUE TITLE
Anti-Neuronal Nuclear Antibody type-3 

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10544]
**Preferred term label:**

**Synonyms**
Anti-DACH1
Anti-Neuronal Nuclear Antibody-type 3 (ANNA3)
Anti-Dachshund-homolog 1 autoantibody

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against Dachshund-homolog 1 (DACH1) autoantigen

PMID: 35150165 The antigen specificity of Anti-Neuronal Nuclear Antibody-type 3 (ANNA3)-IgG is unknown. We identified Dachshund-homolog 1 (DACH1) as the ANNA3 autoantigen and confirmed it by antigen-specific assays, immunohistochemical colocalization and immune absorption experiments.

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:624166 Non-specific autoimmune supratentorial encephalitis with characteristic antibodies
ORPHA:624259 Non-specific autoimmune cerebellar ataxia with characteristic antibodies

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 

Adding new term: 
- ID: HP:6000877
- label: Anti-dachshund family transcription factor 1 antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:35150165
- def: The presence of autoantibodies in the serum that react against dachshund-homolog 1 (DACH1) autoantigen.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10544
- synonyms: Anti-neuronal nuclear antibody-type 3 positivity(NONE)
